dev/com.ibm.ws.ssl/resources/com/ibm/ws/ssl/resources/ssl.nlsprops

```diff
@@ -608,3 +608,7 @@ ssl.defaultKeyStore.expected.CWPKI0817A.useraction=If SSL is not required, this
 ssl.defaultSSLConfig.noSuchKeyStore.CWPKI0818E=CWPKI0818E: The default SSL configuration references a <keyStore> element with an id value of {0}, which does not exist in the configuration or whose definition is missing a password. Either define a keystore with id {0} or update the <ssl id="defaultSSLConfig" keyStoreRef="{0}" /> to reference an existing keystore.
 ssl.defaultSSLConfig.noSuchKeyStore.CWPKI0818E.explanation=The default SSL configuration expects a keystore element, which does not exist.
 ssl.defaultSSLConfig.noSuchKeyStore.CWPKI0818E.useraction=Review the configuration and either change the configuration to reference an existing keystore, or define the referenced keystore.
+#--------------------------------------------------------------------------------------------------
+ssl.defaultKeyStore.not.created.CWPKI0819I=CWPKI0819I: The default keystore will not be created because no password is configured on <keyStore id="defaultKeyStore"/> and no 'keystore_password' environment variable is set. 
```

Comment:
If it makes sense in context, I'd say, "The default keystore is not created..." Use present tense instead of future.
I'd also specify what <keyStore id="defaultKeyStore"/> is, such as "the <keyStore id="defaultKeyStore"/> element".
Place a comma before "and" when you say ",and no 'keystore_password'...".
These comments apply to line 613 as  well.